src/raft/raft.rs

```diff
@@ -750,13 +791,22 @@ impl<T: Storage> Raft<T> {
             } else {
                 m.get_from()
             };
-            info!("{} [term: {}] received a {:?} message with higher term from {} [term: {}]",
-                  self.tag,
-                  self.term,
-                  m.get_msg_type(),
-                  m.get_from(),
-                  m.get_term());
-            self.become_follower(m.get_term(), leader_id);
+            // Never change our term in response to a pre_vote
```

Comment:
The position of this comment is not appropriate.
It's preferred to keep the code structure(and comments as well) the same of the original PR.